src/android/ConnectionSettings.java

```diff
@@ -195,8 +195,21 @@ public static String getGoogleWebAppClientID(Context ctxt) {
             Log.e(ctxt, TAG, "Got exception while retrieving connection settings");
             Log.exception(ctxt, TAG, e);
             return null;
+	    }
 	}
-	}
+
+    public static String get(Context ctxt, String key) {
```

Comment:
Maybe add some documentation that explains what this does, or change the method name?
Right now, it sounds like you can get any key from the settings. But actually, this returns a key from the auth config for this native platform.

So maybe `getNativeAuthValue` or `getAuthValue` instead?